learning-team/2020-08/2020-08-06.md

```diff
@@ -0,0 +1,81 @@
+# Month Day, Year
+
+Facilitator & Note Taker: 
+
+## Attendees
+
+Add yourself to the list if you attend and check the box!
+
+### Core Team
+- [ ] Amy Lam (AL)
+- [ ] Chris Manson (CM)
+- [ ] Isaac Lee (IL)
+- [ ] Jared Galanis (JG)
+- [ ] Jen Weber (JW)
+- [ ] Jessica Jordan (JJ)
+- [ ] Kenneth Larsen (KL)
+- [ ] Melanie Sumner (MS)
+- [ ] Ricardo Mendes (RM)
+- [ ] Robert Wagner (RW)
+
+### Guests
+- [] Your name here (INITIALS)
+
+## Weekly review
+
+### Review action items (weekly)
+See: [Action items board](https://github.com/orgs/ember-learn/projects/47)
+
+### Review Initiatives board (every other week)
+See: [Initiatives board](https://github.com/orgs/ember-learn/projects/19)
+
+### Review PR queue
+See: [tracking sheet](https://docs.google.com/spreadsheets/d/1sPyN9z9wZMpTNwqCfa6R9QSPZkIW4iQd-H4gZC7ILLk/edit#gid=2035777454)
+
+## Topics
+
+<!-- If you would like to add a topic to the agenda please add a suggestion to the PR using the following format: -->
+<!-- ### Your topic (INITIALS, expected duration in minutes) -->
+<!-- replace with topic -->
+### Reorganizing `Built-In Components` in Ember Guides (IL, 10-15 min)
+
+Related links:
+
+- https://github.com/ember-learn/guides-source/issues/1485
+- https://discuss.emberjs.com/t/ember-octane-and-2-way-binding-what-are-you-recommend/18092
+- https://github.com/emberjs/ember.js/blob/master/packages/%40ember/-internals/glimmer/lib/templates/input.hbs (used by `<Input>` component)
+- https://github.com/emberjs/ember.js/blob/master/packages/%40ember/-internals/glimmer/lib/templates/component.hbs (used by `<Textarea>` component)
+
+It wasn't clear to the issuer that event handling for built-in `<Input>` component needs to be different for checkbox and non-checkbox inputs. Our checkbox examples show first and last names (not the best example of a checkbox).
+
+Proposed todos:
+
+- Rename the section from `Built-In Components` to `Input Components`
+- Reorganize the section to:
+  ```
+  Input Components
+  │
+  ├── Native Inputs
+  │
+  └── Built-In Inputs
+      │
+      ├── Non-Checkbox Input
+      │
+      ├── Checkbox Input
+      │
+      └── Textarea
+  ```
+- Check that the event names are correct for the built-in non-checkbox and checkbox input components
+<!-- replace with topic -->
+<!-- replace with topic -->
+<!-- replace with topic -->
```

Comment:
```suggestion
#### Related links

- https://github.com/ember-learn/guides-source/issues/1485
- https://discuss.emberjs.com/t/ember-octane-and-2-way-binding-what-are-you-recommend/18092
- https://github.com/emberjs/ember.js/blob/master/packages/%40ember/-internals/glimmer/lib/templates/input.hbs (used by `<Input>` component)
- https://github.com/emberjs/ember.js/blob/master/packages/%40ember/-internals/glimmer/lib/templates/component.hbs (used by `<Textarea>` component)

#### Problems

There are 3 problems that I'd like us to examine:

It wasn't clear to the issuer that event handling for built-in `<Input>` component needs to be different for checkbox and non-checkbox inputs (camel-cased versus dasherized). Since the underlying implementation for these inputs is different, we could have separate sub-subsections for checkbox and non-checkbox inputs.

Our checkbox examples show first and last names (not the best example of a binary/ternary attribute). They will need to be updated to help address https://github.com/ember-learn/guides-source/issues/1480.

I don't think we currently document how to use the `{{on}}` modifier for native inputs? It'd be good to have a subsection for native inputs for people who want to practice one-way data flow and who want to use `fillIn` test helper to test native inputs.

#### Proposed todos

- Rename the section from `Built-In Components` to `Input Components`
- Reorganize the section to:
  ```
  Input Components
  │
  ├── Native Inputs
  │
  └── Built-In Inputs
      │
      ├── Non-Checkbox Input
      │
      ├── Checkbox Input
      │
      └── Textarea
  ```
- Check that the event names are correct for the built-in non-checkbox and checkbox input components
<!-- replace with topic -->
<!-- replace with topic -->
<!-- replace with topic -->
```